pizza_cutter/metadetect/run_metadetect.py

```diff
@@ -21,9 +21,9 @@ def _make_output_array(
         orig_start_row, orig_start_col, position_offset, wcs):
     arr = eu.numpy_util.add_fields(
                 data,
-                [('id', 'i8'), ('mcal_step', 'S7'),
+                [('slice_id', 'i8'), ('mcal_step', 'S7'),
                  ('ra', 'f8'), ('dec', 'f8')])
```

Comment:
We should def be computing ra, dec from the unsheared positions too once we have the code in place in metadetect. Not needed here though.

Reply:
agreed, I opened a separate issue